src/mapd-con-es6.js

```diff
@@ -1128,12 +1128,18 @@ class MapdCon {
       this._sessionId[0],
       tableName,
       (error, fields) => {
-        if (fields) {
-          const rowDict = fields.row_desc.reduce((accum, value) => {
-            accum[value.col_name] = value
-            return accum
-          }, {})
-          callback(null, this.convertFromThriftTypes(rowDict))
+        if (error) {
```

Comment:
@jrajav Will the `else` ever get hit with this condition in place?

Reply:
Referring to the outer `else` on ln 1143

Reply:
Ok. If there is no error for that case, then ` + error` could be removed from ln 1144.

Reply:
Good call, thanks! Fixed with latest commit